addons/project/views/project_views.xml

```diff
@@ -899,13 +902,23 @@
                                 </div>
                                 <div class="o_kanban_record_bottom mt-3">
                                     <div class="oe_kanban_bottom_left">
-                                        <div class="o_project_kanban_boxes">
+                                        <div class="o_project_kanban_boxes d-flex align-items-baseline">
                                             <a class="o_project_kanban_box" name="action_view_tasks" type="object">
                                                 <div>
                                                     <span class="o_value"><t t-esc="record.task_count.value"/></span>
                                                     <span class="o_label"><t t-esc="record.label_tasks.value"/></span>
                                                 </div>
                                             </a>
+                                            <a groups='project.group_project_milestone' t-if="record.milestone_count and record.milestone_count.value &gt; 0"
+                                                class="o_kanban_inline_block text-muted small"
+                                                name="action_get_list_view"
+                                                type="object"
+                                                attrs="{'invisible': [('allow_milestones', '=', False)]}"
```

Comment:
maybe we could also add `allow_milestones` in the t-if instead of having the attrs. What do you think? Could you test?